ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/api/InterpretationCostBasedLedgerWriterChooserSpec.scala

```diff
@@ -17,6 +18,9 @@ class InterpretationCostBasedLedgerWriterChooserSpec
     with Matchers
     with MockitoSugar
     with ArgumentMatchersSugar {
+
+  private implicit val telemetryContext: TelemetryContext = NoOpTelemetryContext
```

Comment:
This is going to break tests here and elsewhere.

You've got some errors in your latest build along these lines:

```
  org.mockito.exceptions.misusing.InvalidUseOfMatchersException: Invalid use of argument matchers!
```

This is because you're implicitly passing the telemetry context in your mock setups.

```scala
      when(mockWriterCheap.commit(any[String], any[Raw.Envelope], any[CommitMetadata])/* implicit passed here */)
.thenReturn(Future.successful(Acknowledged))
```

You need to use an explicit matcher:

```scala
      when(mockWriterCheap.commit(any[String], any[Raw.Envelope], any[CommitMetadata])(any[TelemetryContext]))
.thenReturn(Future.successful(Acknowledged))
```